homework_12/src/components/formInput/FormInput.js

```diff
@@ -0,0 +1,31 @@
+
+class FormInput extends Component {
+  state = {
+    print: false,
+    value: 'Hello world!'
+  }
+
+  onShowInput = () => {
+    this.setState(prev => ({ print: !prev.print }));
+  }
+
+  getValue = (event) => {
+    this.setState({ value: event.target.value });
+  }
+
+  render() {
+    const { value, print } = this.state;
+
+    return (
+      <>
+        {
+          print
+            ? <input type="text" value={value} onChange={this.getValue} onBlur={this.onShowInput} autoFocus={true} />
```

Comment:
`autoFocus={true}` is it necessary directly set `true`, just `autoFocus` not enough?